packages/python-django-storages/python-django-storages.spec

```diff
@@ -32,7 +32,39 @@ Obsoletes:      python3.11-%{pypi_name} < %{version}-%{release}
 %description
 %{summary}
 
+%package -n python%{python3_pkgversion}-%{pypi_name}+s3
+Summary: Metapackage for python%{python3_pkgversion}-pulpcore: s3 compatible extra
+Requires: python%{python3_pkgversion}-boto3 >= 1.4.4
 
+%description -n python%{python3_pkgversion}-%{pypi_name}+s3
+This is a metapackage bringing in filecache extra requires for python%{python3_pkgversion}-%{pypi_name}
+It contains no code, just makes sure the dependencies are installed.
+
+%files -n python%{python3_pkgversion}-%{pypi_name}+s3
+%ghost %{python3_sitelib}/%{src_name}-%{version}.dist-info/
```

Comment:
Why is this a ghost? Looks like it's included
```suggestion
%{python3_sitelib}/%{src_name}-%{version}.dist-info/
```

Reply:
Yes it's, it took me 2 or 3 days to understand why my packages needed the `ghost` macro

Reply:
Then I'm glad I asked then :)